src/Components/Header.js

```diff
@@ -0,0 +1,31 @@
+// App Header
+import Navigation from "./Navigation.js";
+
+const Header = (prop) => {
```

Comment:
I mentioned this already, but great use of props and components to keep your app.js nice and clear